bin/einhorn

```diff
@@ -181,6 +181,20 @@ EOF
   end
 end
 
+
+BIND_PATTERN = /\A
+  # we accept two types of socket address
+  (?:
+    # ip, host:port
+    (?:(?<host>[^:]+):(?<port>\d+)) |
+    # unix socket path
+    (?<path>[^,:]+)
+  )
+
+  # flags are optional, comma delimited, and come at the end
+  (?<flags>(?:,\w+)*)
```

Comment:
:no_bell: this regex just makes me feel uncomfortable. I feel like it should start with a slash, or be restricted to less characters.